src/kv/test/kv_test.cpp

```diff
@@ -1138,7 +1137,7 @@ TEST_CASE("Deserialise return status")
     REQUIRE(success == kv::CommitResult::SUCCESS);
 
     REQUIRE(
-      store.apply(data, ConsensusType::CFT)->execute() ==
+      store.apply(data, ConsensusType::CFT)->apply() ==
```

Comment:
On the one hand, I'm in favour of the function that returns an `ApplyResult` being called `apply()`. But `apply()->apply()` is very odd. Should the first be `parse()` or `deserialise()` or something?